up-linux-streamer/src/main.rs

```diff
@@ -43,12 +46,26 @@ async fn main() -> Result<(), UStatus> {
         )
     })?;
 
+    let subscription_path = config.usubscription_config.file_path;
+    let usubscription = Arc::new(USubscriptionStaticFile::new(subscription_path));
+
     let mut streamer = UStreamer::new(
         "up-linux-streamer",
         config.up_streamer_config.message_queue_size,
+        usubscription,
     );
 
-    let zenoh_config = ZenohConfig::default();
+    let mut zenoh_config = ZenohConfig::default();
+
+    // Specify the address to listen on using IPv4
+    let ipv4_endpoint = ZenohEndpoint::from_str(config.zenoh_transport_config.endpoint.as_str());
```

Comment:
Should we be able to accept a complete Zenoh config here in zenoh_transport_config?

i.e. able to write the whole content of zenoh_transport_config into zenoh_config.

This feels very narrow.

Reply:
yeah... I mean that would be good. I need to learn what the rest of the zenoh configs are then :)